docs/reference/release-notes/7.7.asciidoc

```diff
@@ -1,22 +1,612 @@
+
+Infra/Packaging::
+* Handle special characters and spaces in JAVA_HOME path in elasticsearch-service.bat {pull}52676[#52676]
+* Limit _FILE env var support to specific vars {pull}52525[#52525] (issue: {issue}52503[#52503])
+* Always set default ES_PATH_CONF for package scriptlets {pull}51827[#51827] (issues: {issue}50246[#50246], {issue}50631[#50631])
+
+Infra/Plugins::
+* Ensure only plugin REST tests are run for plugins {pull}53184[#53184] (issues: {issue}52114[#52114], {issue}53183[#53183])
+
+Machine Learning::
+* Fix node serialization on GET df-nalytics stats without id {pull}54808[#54808] (issue: {issue}54807[#54807])
+* Allow force stopping failed and stopping DF analytics {pull}54650[#54650]
+* Take more care that normalize processes use unique named pipes {pull}54636[#54636] (issue: {issue}43830[#43830])
+* Do not fail Evaluate API when the actual and predicted fields' types differ {pull}54255[#54255] (issue: {issue}54079[#54079])
+* Get ML filters size should default to 100 {pull}54207[#54207] (issues: {issue}39976[#39976], {issue}54206[#54206])
+* Introduce a "starting" datafeed state for lazy jobs {pull}53918[#53918] (issue: {issue}53763[#53763])
+* Only retry persistence failures when the failure is intermittent and stop retrying when analytics job is stopping {pull}53725[#53725] (issue: {issue}53687[#53687])
+* [Inference] fix number inference models returned in x-pack info API {pull}53540[#53540]
+* Make classification evaluation metrics work when there is field mapping type mismatch {pull}53458[#53458] (issue: {issue}53485[#53485])
+* Perform evaluation in multiple steps when necessary {pull}53295[#53295]
+* Specifying missing_field_value value and using it instead of empty_string {pull}53108[#53108] (issue: {issue}1034[#1034])
+* Use event.timezone in ingest pipeline from find_file_structure {pull}52720[#52720] (issue: {issue}9458[#9458])
+* [Inference] don't return inflated definition when storing trained models {pull}52573[#52573]
+* Validate tree feature index is within range {pull}52460[#52460]
+* Better error when persistent task assignment disabled {pull}52014[#52014] (issue: {issue}51956[#51956])
+* Fix possible race condition starting datafeed {pull}51646[#51646] (issues: {issue}50886[#50886], {issue}51302[#51302])
+* Fix 2 digit year regex in find_file_structure {pull}51469[#51469]
+* Fix possible race condition when starting datafeed {pull}51302[#51302] (issue: {issue}51285[#51285])
+* Validate classification dependent_variable cardinality is at lea… {pull}51232[#51232]
+* Do not copy mapping from dependent variable to prediction field in regression analysis {pull}51227[#51227]
+* Address two edge cases for categorization.GrokPatternCreator#findBestGrokMatchFromExamples {pull}51168[#51168]
+* Calculate results and snapshot retention using latest bucket timestamps {pull}51061[#51061]
+
+Mapping::
+* Disallow duplicate `dynamic_templates` names {pull}51817[#51817] (issue: {issue}28988[#28988])
```

Comment:
Looks like this was closed but not merged. It's superseded by #52062, which is still open.
```suggestion
```